### PR TITLE
[#87255752] Retrieve task status

### PR DESCRIPTION
**Pull request by datianshi**

### CODE REVIEW

File: bosh/bosh_test.go [hunk 1]
```diff
@@ -133,6 +133,49 @@ var _ = Describe("Bosh", func() {
 				Ω(capturedEntity.ContentType).Should(Equal("text/yaml"))
 			})
 		})
+		Describe("Bosh Retrieve Task staus", func() {
+			var ()
```
Comment (malston) [inline bosh/bosh_test.go]: Remove empty var

File: bosh/bosh_test.go [hunk 2]
```diff
@@ -133,6 +133,49 @@ var _ = Describe("Bosh", func() {
...
+			Context("Http Request Failure", func() {
+				It("Should return error", func() {
+					httpRequestSuccessful = false
```
Comment (malston) [inline bosh/bosh_test.go]: Why is this flag here?

File: bosh/task.go [hunk 3]
```diff
@@ -21,3 +30,19 @@ func retrieveTaskId(resp *http.Response) (taskId int, err error) {
 	idString := regex.ReplaceAllString(redirectUrls[0], "")
 	return strconv.Atoi(idString)
 }
+
+func retrieveTaskStatus(resp *http.Response) (task *Task, err error) {
+	if resp.StatusCode != 200 {
+		err = errors.New("Retrieving Task status failed")
```
Comment (malston) [inline bosh/task.go]: Can you provide a reason for failure?

File: bosh/bosh_test.go (hunk 1 above)
Comment (datianshi) [inline bosh/bosh_test.go]: Will do

File: bosh/bosh_test.go (hunk 2 above)
Comment (datianshi) [inline bosh/bosh_test.go]: To make sure the http request succeeds

File: bosh/task.go (hunk 3 above)
Comment (datianshi) [inline bosh/task.go]: You mean more details of error message?

**Pull request merged**